third-party/qthread/qthread-src/src/affinity/binders.c

```diff
@@ -91,9 +91,15 @@ int INTERNAL qt_affinity_compact(int num_workers,
 void INTERNAL qt_affinity_init(qthread_shepherd_id_t *nbshepherds,
                                qthread_worker_id_t   *nbworkers,
                                size_t                *hw_par)
-{                           
-  hwloc_topology_init(&topology);
-  hwloc_topology_load(topology);
+{                      
+#ifdef HWLOC_GET_TOPOLOGY_FUNCTION
+  extern hwloc_topology_t HWLOC_GET_TOPOLOGY_FUNCTION;
```

Comment:
I think technically `chpl_topo_getHwlocTopology` returns `void*`. I don't think the different declared types will hurt anything, but wanted to check that you agree.

Reply:
Fixed.